changelog/1390.doc.rst

```diff
@@ -0,0 +1 @@
+Describe `plasma_calculator` in narrative documentation
```

Comment:
```suggestion
Described the |plasma-calculator| in the narrative documentation.
```
This includes some pretty minor edits for consistency with other changelog entries, and applies a [reST substitution](https://docs.plasmapy.org/en/latest/development/doc_guide.html#substitutions).